content/articles/2017-08-tidyr-0.7.0.Rmd

```diff
@@ -0,0 +1,209 @@
+---
+title: tidyr 0.7.0
+slug: tidyr-0.7.0
+description: >
+  The next installment of tidyr is finally on CRAN! This version
+  brings tidy eval to a crucial component of the tidyverse workflow.
+author: Lionel Henry
+date: 2017-08-15
+photo:
+  url: https://unsplash.com/photos/8tem2WpFPhM
+  author: Radek Grzybowski
+categories: [packages]
+---
+
+```{r setup, include = FALSE}
+library("tidyr")
+library("dplyr")
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are happy to announce that tidyr 0.7.0 is now available on CRAN.
+This version is mostly about ensuring consistent semantics across the
```

Comment:
Let's rearrange slightly:

There are two big changes:

* tidyr now supports tidy evaluation (or tidy eval for short) ...

* tidyr uses tidyselect as selection backend ... 

These changes will probably only affect your code in minor ways but help improve consistency across the tidyverse.